compiler/packages/react-compiler-healthcheck/src/checks/strictMode.ts

```diff
@@ -11,6 +11,9 @@ const JsFileExtensionRE = /(js|ts|jsx|tsx)$/;
 const StrictModeRE = /<(React\.StrictMode|StrictMode)>/;
 let StrictModeUsage = false;
 
+const NextJSConfigRE = /next\.config\.js/;
```

Comment:
The Next config file can end with `.mjs` extension too, according to [the documentation](https://nextjs.org/docs/app/api-reference/next-config-js).

So maybe we can check for that as well ?
```suggestion
const NextJSConfigRE = /next\.config\.m?js/;
```